.gitignore

```diff
@@ -10,6 +10,13 @@ lerna-debug.log*
 # Diagnostic reports (https://nodejs.org/api/report.html)
 report.[0-9]*.[0-9]*.[0-9]*.[0-9]*.json
 
+# Ignore a specific file
+
+menubook.css
```

Comment:
This file is used in your code

Reply:
ok so I was thinking that if I use this file in the separate branch (because menu.css is used on another page on another branch) will it create some sort of conflict? I mean if I have the menu branch pushed that already has this menubook.css file in there, do I still have to leave it when working with a separate branch?